app/main.py

```diff
@@ -1,8 +1,18 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
 
+    def __init__(self, name, age):
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+def create_person_list(people_list):
+    persons = []
+    for person in people_list:
+        new_person = Person(person["name"], person["age"])
+        persons.append(new_person)
+    for person in people_list:
+        person.get("wife") and setattr(Person.people[person["name"]], "wife", Person.people[person["wife"]])
+        person.get("husband") and setattr(Person.people[person["name"]], "husband", Person.people[person["husband"]])
```

Comment:
The use of `setattr` and accessing class variable `people` here makes the code hard to read and understand. It would be better to create a method in the `Person` class to set the `wife` and `husband` attributes. Also, you're iterating over `people_list` twice, which is inefficient. You can set the `wife` and `husband` attributes in the first loop.